### DISCUSSION TITLE
Impassa SCW9057G-433

### DISCUSSION THREAD

**Discussion by zrlram** [taligentx/dscKeybusInterface#204] [dscKeybusInterface Discussion]
I have an Impassa SCW9057G-433 that looks just like the other DSC panels. I was wondering if anyone has successfully connected to one of these panels? I saw the 9047 series is supported, but no word on the 9057.

Also, has anyone managed to intercept the 433 wireless protocol and written a receiver/decoder for it?

Thx

**Comment by kricon** [comment]
Where you saw 9047 as supported? DSC Impassa (just like DSC Alexor) doesn't have any external interface possibility (except PC Link used for programming/IP communicators) and thus it isn't supported by this project. So connection is not possible, only DSC Classic and PowerSeries is supported, through Keybus interface (hence the name of this project).

EDIT: I see why you throught 9047 is supported, it is called PowerSeries 9047. That was older (and one-way communication only) version prior to 9057 when DSC start to use Impassa naming (and it is 2-way communication with support for Alexor wireless sirens and keypads). User interface/programming and compatible wireless modules is same for all PowerSeries/Alexor/Impassa but that's where similarities ends, thoose self-contained and wireless systems isn't supported.

Your best bet if you want basic remote arming/disarming or alarm notification is to use onboard PGM outputs (it can be set to output when system is armed/in alarm etc) and Zone inputs (it can be set to dis/arm the system) with custom-programmed hardware.

About wireless protocol, as far as I know, 2-way RF panel wouldn't transmit anything if not requested by enrolled WT5500 (wireless lcd keypad) or 2-way keyfob (which only receives basic armed/disarmed/trouble states) and its communications are encrypted. I think one-way sensors can be sniffed (after extensive testing) but they only transmit after opening/closing or low battery/tamper condition and you're likely to trip RF jamming detection.
